assets/css/_late_view.scss

```
@@ -10,8 +10,8 @@ $late-view-background-color: $color-bg-light;
   top: 0;
   z-index: map-get($z-page-layout-context, "view");
 
-  @media screen and (max-width: 700px) {
-    width: 100%;
+  @media screen and (max-width: map-get($breakpoints, "max-mobile-landscape-tablet-portrait-width")) {
+    width: 100vw;
```

suggestion: what do you think about using `inset: 0` instead?
```suggestion
    inset: 0;
```

---

So unfortunately in part because of how the Late View page is implemented (read: not very responsive) this doesn't actually result in it cleanly taking up the full page (I don't know how to best describe it but you can try it out quickly and see). There's also the matter that we want it to cover the nav, but `inset: 0` would just cause it to take up the full size of its container which is the app container element.

---

ah right, we'd need to also add `position: fixed;` to make it use the viewport as it's target.

Part of me worries about using `100vw` if the panel could accidentally get moved horizontally somehow, so I would _prefer_ that we use `fixed` and `inset`, but I'm considering this non-blocking and up to you.